card.txt

```diff
@@ -0,0 +1 @@
+This is our great feature. It's awesome!
```

Comment:
@ratxt1 and @cyhk  - please add more text...this is JUST NOT ENOUGH!!!

Reply:
@elie changes made in most recent commit.

Reply:
LGTM!